releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,46 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.0-preview-010550001-251 - November 22, 2019
+* Added the following method to connect to the configuration store using Azure Active Directory.
+   ````csharp
+   public AzureAppConfigurationOptions Connect(Uri endpoint, TokenCredential credential)
+   ````
+* Removed the `ConnectWithManagedIdentity` method to connect to the configuration store using system-assigned managed identity. This now requires adding a reference to the package [`Azure.Identity`](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity) and updating the code to use the `Connect` method.
+
+   #### Before
+   ````csharp
+   IConfigurationBuilder configBuilder = new ConfigurationBuilder();
+   IConfiguration configuration = configBuilder.AddAzureAppConfiguration(options => {
+      options.ConnectWithManagedIdentity(endpoint);
+   }).Build();
+   ````
+   #### After
+   ````csharp
+   IConfigurationBuilder configBuilder = new ConfigurationBuilder();
+   IConfiguration configuration = configBuilder.AddAzureAppConfiguration(options => {
+      options.Connect(endpoint, new ManagedIdentityCredential());
+   }).Build();
+   ````
```

Comment:
One another thing we should mention here is the user needs to grant whatever the identity (managed identity included) the data plane reader role access (instead of the contributor role) for this to work.